ci/unit_tests/dialog_json2xml/main_test.py

```diff
@@ -44,13 +46,16 @@ def test_mainValidActions(self):
 
         self.t_noException([[inputJsonPath, '-d', outputXmlDirPath]])
 
-        expectedXml = ""
-        outputXml = ""
-
         with open(expectedXmlPath, 'r') as expectedXmlFile:
-            expectedXml = expectedXmlFile.read()
+            expectedXml = etree.XML(expectedXmlFile.read(), etree.XMLParser(remove_blank_text=True))
+            for parent in expectedXml.xpath('//*[./*]'): # Search for parent elements
+                print(parent)
```

Comment:
Remove this print to avoid std-out spoiling

Reply:
ok, thanks :)